command.c

```diff
@@ -381,6 +381,7 @@ void redis_parse_cmd(struct cmd *r) {
             break;
 
         case SW_REQ_TYPE_LEN:
+            // Parse length of request/command
```

Comment:
Better consistently use Redis terminologoy, so "command" here. If we need to specify, how about "command array"?

Reply:
I remove all comments, and instead rename SW_REQ_XXX to SW_CMD_XXX, like SW_CMD_TYPE_LEN which parses the command type length.
I think thats what was a bit confusing to me